catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -226,6 +227,8 @@ public void init() {
         "/forms/:id",
         APPLICATION_JSON,
         (req, res) -> {
+          EventApplication.notifyListeners("searchform");
```

Comment:
should we establish these as constants in the ui api module?

Reply:
done